src/vmemcache.c

```diff
@@ -69,6 +173,61 @@ vmemcache_newU(const char *dir, size_t max_size, size_t fragment_size,
 void
 vmemcache_delete(VMEMcache *cache)
 {
+	while (vmemcache_evict(cache, NULL, 0) == 0)
+		;
+	repl_p_destroy(&cache->repl);
+	ravl_delete(cache->index);
+	vmcache_heap_destroy(cache->heap);
+	munmap(cache->addr, cache->size);
+	Free(cache);
+}
+
+/*
+ * vmemcache_populate_fragments -- (internal) populate fragments
```

Comment:
Describe this a bit better ;)
`Copies value content to heap entries` and so on.

Reply:
Done